draft-ietf-taps-interface.md

```diff
@@ -763,7 +763,7 @@ Setting a Transport Property to a value overrides the previous value of this Tra
 TransportProperties.Set(property, value)
 ~~~
 
-Selection Properties of type `Preference` can be frequently used. Implementations MAY therefore provide additional convenience functions, see {{preference-conv}} for examples.
+Selection Properties of type `Preference` might often be frequently used. Implementations MAY therefore provide additional convenience functions to simplify use, see {{preference-conv}} for examples.
```

Comment:
To me, the original sentence is confusing, and "might often be frequently used" just highlights the strangeness of this more ("often frequently"?). Would it be better to just remove the first sentence altogether? It doesn't seem to add much value anyway.